openbb_platform/core/openbb_core/app/static/container.py

```diff
@@ -1,15 +1,43 @@
 """Container class."""
 
-from typing import Any, Optional, Tuple
+from concurrent.futures import ThreadPoolExecutor
+from typing import Any, Dict, Optional, Tuple
+
+from requests import get
 
 from openbb_core.app.command_runner import CommandRunner
+from openbb_core.app.constants import REPOSITORY_URL
 from openbb_core.app.model.abstract.error import OpenBBError
 from openbb_core.app.model.obbject import OBBject
 
 
 class Container:
     """Container class for the command runner session."""
 
+    _openbb_providers: Dict[str, str] = {}
+    _executor = ThreadPoolExecutor()
+
+    @classmethod
+    def _initialize_providers(cls):
+        """Fetch providers maintained in OpenBB repository and cache them."""
+        try:
+            response = get(
```

Comment:
What is the purpose of us constantly updating `reference.json` in the repo if we don't use this file, which actually contains all the information we need to know?

Reply:
It doesn't contain all the information you need to know, if we are telling the user to install it, it's because it's not found locally. You're misunderstanding the purpose of `reference.json`

Agree it might be too much effort for a msg, this follows up @jmaslek of being more specific with the message, lmk if it's overkill

Reply:
We could essentially get the same effect by changing this message to "is not installed" from 'not found'. We could also operate under the assumptions that OpenBB extensions need to begin with `openbb-{package-name}`.